Extending/Property-Editors/package-manifest.md

```diff
@@ -228,6 +233,13 @@ The parameter editors array follows the same format as the property editors desc
 }
 ```
 
+## Bundling
+`bundleOptions` is an enum that expects one of the following values:
```

Comment:
I don't know the suggested changes here makes sense, but `enum` threw me off a bit 🤔 
Perhaps we could loose the ´type´?

```suggestion
`bundleOptions` is an enumerable type that expects one of the following values:
```